e2e/internal/e2e/docker.go

```diff
@@ -50,6 +51,13 @@ func PrepRegistry(t *testing.T, env TestEnv) {
 			ExpectExit(0),
 		)
 
+		// sleep a bit to let time to registry to start
+		// and setup everything. The start script already
+		// sleep for 2 seconds before pushing a busybox image in
```

Comment:
It looks like its sleeping for 5 seconds

```suggestion
		// sleep for 5 seconds before pushing a busybox image in
```

Reply:
Oh I refer to this line https://github.com/sylabs/singularity/blob/master/e2e/testdata/Docker_registry.def#L19

Reply:
I will mention the file to avoid confusion